corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +47,12 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_web_user_usercases_if_applicable(user, domain):
+    domain_obj = Domain.get_by_name(domain)
+    if USH_WEB_USER_CASE_CREATION.enabled(domain):
+        sync_usercases_task.delay(user._id, domain)
```

Comment:
this should still check the domain config if the FF is enabled:
```suggestion
    if not USH_WEB_USER_CASE_CREATION.enabled(domain):
        return
    domain_obj = Domain.get_by_name(domain)
    if domain_obj.usercase_enabled:
        sync_usercases_task.delay(user._id, domain)
```